### ISSUE TITLE
Cannot Compile on m1 mac

### ISSUE THREAD

**Issue by lecoqjacob** [EngoEngine/engo#788]
<img width="598" alt="CleanShot 2022-05-21 at 09 43 17@2x" src="https://user-images.githubusercontent.com/9278174/169656700-e1f3013a-c8f1-4eb1-a6af-23dbc459b565.png">

Im using m1 mac on the latest `Monterey`, and I keep getting erros where `GamePad` is not available to me.

<img width="1213" alt="CleanShot 2022-05-21 at 09 44 13@2x" src="https://user-images.githubusercontent.com/9278174/169656723-6618a029-8b21-4fb2-a782-e9eb2bbf36e2.png">

Looking at the source I believe it is not getting compiled due to my arch?

**Comment by KennethGrace** [comment]
I'd like to add to this post and add that there is an additional issue in `EngoEngine/gl`. Your last compiler error there,
```
../../../../../go/pkg/mod/github.com/!engo!engine/engo@v1.0.6/engo_mobile.go:97:25: too many arguments in call to gl.NewContext
	have (interface{})
	want ()
```
seems to be caused by a mismatch in the build tags between `engo_mobile.go` and `gl_mobile.go`. I've created an [issue](https://github.com/EngoEngine/gl/issues/39) there to track as well.

Hope we can get this working on our M1s soon. 😄

**Comment by KennethGrace** [comment]
looks like this was resolved in [Pull Request #37](https://github.com/EngoEngine/gl/pull/37) in `EngoEngine/gl` and on [Pull Request #785](https://github.com/EngoEngine/engo/pull/785). Neither of these changes have been released yet though, so you'll need to pull from the master branch.

```
go get github.com/EngoEngine/gl@master
go get github.com/EngoEngine/engo@master
```

This solved the issue on my M1, so you should be able to close this issue.

**Comment by otraore** [comment]
The changes were released [here](https://github.com/EngoEngine/engo/releases/tag/1.07).

You should be able to run `go get -u` @lecoqjacob and get the latest version to be able to run the example. 

Let me know if there are any issues.

Closed by https://github.com/EngoEngine/engo/pull/785 being released.
